src/server/plugins/router.ts

```diff
@@ -16,7 +17,7 @@ export default {
           method: 'get',
           path: '/help/get-support/{slug?}',
           async handler(request: Request, h: ResponseToolkit) {
```

Comment:
Just to avoid the `as string` (because it might be `undefined`) you can add:

```suggestion
          async handler(request: Request<{ Params: { slug?: string }}>, h: ResponseToolkit) {
```

Reply:
Whoops! Thanks